python/3.12/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM        --platform=$TARGETOS/$TARGETARCH python:3.12-slim
+FROM        --platform=$TARGETOS/$TARGETARCH python:3.7-slim-bookworm
```

Comment:
@QuintenQVD0 @parkervcp 

A mistake was made here. This is the 3.12 Dockerfile and you are using python:3.7 . This is why my egg is not working